index.js

```diff
@@ -1,17 +1,87 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = []
+    this.length = this.items.length
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item)
+    this.items = this.items.sort((a, b) => a - b)
+    this.length = this.items
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (this.items < 0 || this.items.length === 0 || pos > this.items.length) {
```

Comment:
aqui realmente podríamos hacer una única evaluación -> `if (pos > this.items.length -1)`, ya que this.items es un array no compararía con un número, lo que queremos saber es si hay length o si tenemos algún elemento en el array, esto sería más correcto